kotlinx-coroutines-debug/src/DebugProbes.kt

```diff
@@ -24,16 +25,26 @@ import kotlin.coroutines.*
  * In practice, it means that for snapshotting operations in progress, for each concurrent coroutine either
  * the state prior to the operation or the state that was reached during the current operation is observed.
  *
- * ### Installed hooks
- * * `probeCoroutineResumed` is invoked on every [Continuation.resume].
- * * `probeCoroutineSuspended` is invoked on every continuation suspension.
- * * `probeCoroutineCreated` is invoked on every coroutine creation.
- *
  * ### Overhead
- *  * Every created coroutine is stored in a concurrent hash map and hash map is looked up and
+ *
+ *  - Every created coroutine is stored in a concurrent hash map and hash map is looked up and
  *    updated on each suspension and resumption.
- *  * If [DebugProbes.enableCreationStackTraces] is enabled, stack trace of the current thread is captured on
+ *  - If [DebugProbes.enableCreationStackTraces] is enabled, stack trace of the current thread is captured on
  *    each created coroutine that is a rough equivalent of throwing an exception per each created coroutine.
+ *
+ * ### Internal machinery and classloading.
+ *
+ * Under the hood, debug probes replace internal `kotlin.coroutines.jvm.internal.DebugProbesKt` class with the following
+ * empty static methods:
```

Comment:
```suggestion
 * Under the hood, debug probes replace internal `kotlin.coroutines.jvm.internal.DebugProbesKt` class that has the following
 * empty static methods:
```

Otherwise, we get the somewhat confusing "replace A with B with C."